MatrixStruct/MatrixStruct/ZMatrix2x2.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Policy;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace MatrixStruct
+{
+    public struct ZMatrix2x2
+    {
+        private float[,] Matrix = new float[2, 2]; //закрытое поле
```

Comment:
Закрытые поля/свойства принято называть с маленькой буквы
```suggestion
        private float[,] matrix = new float[2, 2]; //закрытое поле
```